core/conn/jdbc_type2/Makefile

```diff
@@ -57,7 +57,15 @@ T2_OBJS  = $(OUTDIR)/CommonDiags.o \
        $(OUTDIR)/ResStatisticsStatement.o \
        $(OUTDIR)/Vproc.o \
        $(OUTDIR)/type2version.o
-
+CSES := org.trafodion.jdbc.t2.JdbcDebug \
+		org.trafodion.jdbc.t2.DataWrapper \
+		org.trafodion.jdbc.t2.SQLMXCallableStatement \
+		org.trafodion.jdbc.t2.SQLMXConnection \
+		org.trafodion.jdbc.t2.SQLMXDatabaseMetaData \
+		org.trafodion.jdbc.t2.T2Driver \
+		org.trafodion.jdbc.t2.SQLMXPreparedStatement \
+		org.trafodion.jdbc.t2.SQLMXResultSet \
+		org.trafodion.jdbc.t2.SQLMXStatement
```

Comment:
Would this change be needed if we touch the package name in T2 source to include apache ? Should we change the package name (org.apache.trafodion...) in the source and the file layout  to make it consistent ?